package.json

```diff
@@ -44,16 +45,13 @@
     "axios": "^0.25.0",
     "@itwin/core-geometry": "^3.0.0"
   },
-  "peerDependencies": {
-    "@itwin/core-common": "^3.0.0"
```

Comment:
"@itwin/core-common" is where the interface implemented by this client is defined. Is adding it as devDependecy only is enough to ensure proper structure version on the consuming app?

Reply:
Hey @MarcBedard8 , sorry, I thought i had replied earlier. The RealityDataAccess interface is defined in core-common, but its public, so theoretically it shouldnt break for other apps if things were added. If there is a change to the interface, we'd need to bump the version here. In Consuming apps they should already have a direct dep on core-common anyway. because its a peer of core-frontend, and that peer dep req needs to be met.